client/src/components/Burger/Burger.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from "react";
+import "./Burger.css";
+
+// Class Based React Component
+class Burger extends Component {
+  constructor(props) {
+    super(props);
+    console.log(props);
```

Comment:
Can you remove any `console.log` statements before making a PR from now on?

Reply:
Removed.